target/linux/mediatek/dts/mt7981b-asus_rt-ax57go.dts

```diff
@@ -0,0 +1,267 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include "mt7981.dtsi"
+/ {
+	model = "RT-AX57GO";
+	compatible = "asus,rt-ax57go", "mediatek,mt7981";
+	
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-override = "ubi.mtd=UBI_DEV";
+	};
+
+	memory {
+		// fpga ddr2: 128MB*2
+		reg = <0 0x40000000 0 0x10000000>;
+	};
+
+	reserved-memory {
+		/* 24 KiB reserved for PWM[ch0,ch1,ch2] */
+		pwm_reserved: pwmbuf@4C7FA000 {
```

Comment:
Please remove pwm reserved-memory and pinctrl. This device doesn't have pwm peripheral.